lib/httpi/adapter/curb.rb

```diff
@@ -59,10 +59,11 @@ def do_request
       def setup_client
         basic_setup
 
-        if @request.auth.ntlm?
+        if @request.auth.ntlm? && !Curl.ntlm?
           raise NotSupportedError, "curb does not support NTLM authentication"
```

Comment:
It'd be great to tell on this message what curb's version support it, just to help the developer.